config.yml

```diff
@@ -29,7 +31,7 @@ MWI:
   fewsnet_dates: [ "200907","200910","201001","201004","201007","201010","201101","201104","201107","201110","201201",
                    "201204","201210","201301","201304","201307","201310","201401","201404","201407","201410","201501","201504",
                    "201507","201510","201602","201606","201610","201702","201706","201710","201802","201806","201812",
-                   "201902","201906","201910","202002","202006","202008" ]
+                   "201902","201906","201910","202002","202006","202008","202010"]
```

Comment:
How do you envision keeping this config list up-to-date and accurate? Should we pre-populate the scheduled publications for the next year or two? How would we know that a date is "missing"?

Reply:
Those are very good points, that need some dev thought. I added issues for them for now and will try to get back to those issues soonish.